gnomad/gnomadv4_extract_test.py

```diff
@@ -62,13 +62,17 @@
 
 # keep only the rows
 # persist to avoid double evaluation later 
-ht = mt.rows().persist()
+#ht = mt.rows().persist()
+
+# remove persist which is suspected to trigger a cryptic error on the dataproc
```

Comment:
Please remove such obsolete statements, it makes the code really hard to read. You have full history in git.

```suggestion
```